src/main/resources/application-lib.yaml

```diff
@@ -21,7 +21,4 @@ azure:
     container-name: ${CONTAINER_NAME:CONTAINER_NAME}
     blob-url-suffix: ${BLOB_URL_SUFFIX:BLOB_URL_SUFFIX}
 
-
-
-
-
+idempotent-flag-ingestion: ${idempotent-flag:true}
```

Comment:
where is this coming from? KV or launch darkly?

Reply:
It will come from consumer LRD/JRD. If not came default idempotent behaviour.